src/Microsoft.ML.LightGBM.StaticPipe/LightGbmStaticExtensions.cs

```diff
@@ -279,10 +284,10 @@ public static class LightGbmStaticExtensions
         /// <param name="label">The label, or dependent variable.</param>
         /// <param name="features">The features, or independent variables.</param>
         /// <param name="weights">The weights column.</param>
-        /// <param name="numLeaves">The number of leaves to use.</param>
-        /// <param name="numBoostRound">Number of iterations.</param>
-        /// <param name="minDataPerLeaf">The minimal number of documents allowed in a leaf of the tree, out of the subsampled data.</param>
+        /// <param name="numberOfLeaves">The number of leaves to use.</param>
+        /// <param name="minimumDataPerLeaf">The minimal number of documents allowed in a leaf of the tree, out of the subsampled data.</param>
```

Comment:
```suggestion
        /// <param name="minimumDataPerLeaf">The minimal number of data points allowed in a leaf of the tree, out of the subsampled data.</param>
```

Reply:
I am using minimumExampleCountPerLeaf based on the previous comments. 

---
In reply to: [261458306](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261458306) [](ancestors = 261458306)

Reply:
Also replaced all comments with what you provided above.

---
In reply to: [261777274](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261777274) [](ancestors = 261777274,261458306)